manifests/m/Microsoft/SqlPackage/162.3.566/Microsoft.SqlPackage.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.SqlPackage
+PackageVersion: 162.3.566
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://learn.microsoft.com/en-us/sql/tools/sqlpackage/sqlpackage?view=sql-server-ver16
+Author: Microsoft Corporation
+PackageName: Microsoft SqlPackage
+PackageUrl: https://learn.microsoft.com/en-us/sql/tools/sqlpackage/sqlpackage?view=sql-server-ver16
+License: Proprietary
+Copyright: Copyright (c) Microsoft Corporation
+ShortDescription: SqlPackage is a command-line utility that automates the database development tasks by exposing some of the public Data-Tier Application Framework (DacFx) APIs.
+Description: qlPackage is a command-line utility that automates the database development tasks by exposing some of the public Data-Tier Application Framework (DacFx) APIs. The primary use cases for SqlPackage focus on database portability and deployments for the SQL Server, Azure SQL, and Azure Synapse Analytics family of databases. SqlPackage can be automated using Azure Pipelines and GitHub actions or other CI/CD tools.

```

```suggestion
Description: SqlPackage is a command-line utility that automates the database development tasks by exposing some of the public Data-Tier Application Framework (DacFx) APIs. The primary use cases for SqlPackage focus on database portability and deployments for the SQL Server, Azure SQL, and Azure Synapse Analytics family of databases. SqlPackage can be automated using Azure Pipelines and GitHub actions or other CI/CD tools.
```

Common copy-paste error.